app/api/Torrent.js

```diff
@@ -23,6 +21,8 @@ export default class Torrent {
 
   server: Object;
```

Comment:
Ideally, `server: Object`should be `server: WebTorrent`. Not sure why I didn't do that in the first place 😂